.travis.yml

```diff
@@ -3,35 +3,33 @@ language: python
 services:
     - mongodb
 
-matrix:
-  fast_finish: true
+jobs:
   include:
-    - env: TOXENV=flake8
+    - stage: pretest
+      env: TOXENV=flake8
       python: 3.6
     - env: TOXENV=pylint
       python: 3.6
     - env: TOXENV=doc8
       python: 3.6
-    - env: TOXENV=packaging
-      python: 3.6
     - env: TOXENV=docs
       python: 3.6
-    - env: TOXENV=py34
+    - stage: test
```

Comment:
Oh! :heart:

Reply:
This also makes it run each group completely before moving on to the next.  If a part of the group fails, it skips the following groups.